sympde/topology/domain.py

```diff
@@ -337,7 +342,7 @@ def from_file( cls, filename ):
         if not(ext == '.h5'):
             raise ValueError('> Only h5 files are supported')
         # ...
-        from sympde.topology.mapping import Mapping, MultiPatchMapping
+        from sympde.topology import BaseMapping
```

Comment:
Please use the full import path whenever possible.
```suggestion
        from sympde.topology.base_mapping import BaseMapping
```